indico/modules/events/contributions/util.py

```diff
@@ -198,6 +199,11 @@ def get_contribution_ical_file(contrib):
     serializer = Serializer.create('ics')
     return BytesIO(serializer(data))
 
+def is_valid_email(email):
+    if len(email) > 7:
+        if re.match("^.+@([?)[a-zA-Z0-9-.]+.([a-zA-Z]{2,3}|[0-9]{1,3})(]?)$", email) != None:
```

Comment:
PEP8 states that you should use `is not` for this comparison:

> Comparisons to singletons like None should always be done with is or is not, never the equality operators.

Also, did you even test this regex? 

```
>>> re.match("^.+@([?)[a-zA-Z0-9-.]+.([a-zA-Z]{2,3}|[0-9]{1,3})(]?)$", 'foo@bar.google')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/adrian/.python-env/lib64/python2.7/re.py", line 141, in match
    return _compile(pattern, flags).match(string)
  File "/home/adrian/.python-env/lib64/python2.7/re.py", line 251, in _compile
    raise error, v # invalid expression
sre_constants.error: unbalanced parenthesis
```

Regardless of this, the regex doesn't make much sense and also doesn't take anything but ASCII into account.